proto/sdk/alpha/alpha.proto

```diff
@@ -138,13 +138,53 @@ service SDK {
 
     // Updates a Counter. Returns NOT_FOUND if the Counter does not exist (name cannot be updated).
     // Returns INVALID_ARGUMENT if the field mask path(s) are not field(s) of the Counter.
+    // Returns OUT_OF_RANGE if the Count is out of range [0,Capacity].
     rpc UpdateCounter(UpdateCounterRequest) returns (Counter) {
       option (google.api.http) = {
         patch: "/v1alpha1/{counter.name=counters/*}"
         body: "counter"
       };
       option (google.api.method_signature) = "counter,update_mask";
     }
+
+    // Gets a List. Returns NOT_FOUND if the List does not exist.
+    rpc GetList(GetListRequest) returns (List) {
+      option (google.api.http) = {
+        get: "/v1alpha1/{name=lists/*}"
+      };
+      option (google.api.method_signature) = "name";
+    }
+
+    // UpdateList returns the updated List. Returns NOT_FOUND if the List does not exist (name cannot be updated).
+    // **THIS WILL OVERWRITE ALL EXISTING LIST.VALUES WITH REQUEST LIST.VALUES** use AddListValue()
+    // or RemoveListValue() for modifying the List.Values field.
+    // Returns INVALID_ARGUMENT if the field mask path(s) are not field(s) of the List.
+    rpc UpdateList(UpdateListRequest) returns (List) {
+      option (google.api.http) = {
+        patch: "/v1alpha1/{list.name=lists/*}"
+        body: "list"
+      };
+      option (google.api.method_signature) = "list,update_mask";
+    }
+
+    // Adds a value to a List and returns updated List. Returns NOT_FOUND if the List does not exist.
+    // Returns ALREADY_EXISTS if the value is already in the List.
+    // Returns OUT_OF_RANGE if the List is already at Capacity.
+    rpc AddListValue(AddListValueRequest) returns (List) {
+      option (google.api.http) = {
+        post: "/v1/{name=lists/*}:addValue"
```

Comment:
Per the AIP yes, but I think this confuses things, since we're not actually passing a list object, we're passing the string name of the list. IMO the naming should be consistent with GetList which also passes a string name. LMK if you think we should follow the AIP in here.